src/components/footer/footer.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Block } from 'baseui/block';
+import { Footer } from '../../types/homepage';
+import {
+  ALIGN,
+  StyledNavigationItem,
+  StyledNavigationList,
+} from 'baseui/header-navigation';
+
+declare type FooterSectionProps = {
+  data: Footer;
+};
+
+export const FooterSection: React.FC<FooterSectionProps> = ({ data }) => {
+  return (
+    <Block backgroundColor={'#f5f5f5'} padding={['scale700']}>
+      <StyledNavigationList $align={ALIGN.left}>
+        <StyledNavigationItem>
+          <Block
+            alignItems="center"
+            height="100%"
+            display="flex"
+            paddingLeft={['0', '0', 'scale1600']}
+          >
+            {`© ${new Date().getFullYear()} ${data.Copyright}`}
```

Comment:
I have given this feedback in writing as well and in person - the year should come from the Strapi. This is the year of incorporation of a company and not the current year.

<img width="960" alt="Screenshot 2023-04-03 at 10 34 42 AM" src="https://user-images.githubusercontent.com/328858/229416243-e4b18ef3-6c1e-4d41-a58c-f1641f3ac815.png">

It would be good to double check your implementation and do self review please to ensure same feedback need not be given twice.